### PR TITLE
Disable Azure.Management.KeyVault live tests

### PR DESCRIPTION
**Pull request by heaths**
Disable until #12211 is fixed.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

File: sdk/keyvault/service.projects [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- Remove mgmt plane tests from nightly live runs: https://github.com/Azure/azure-sdk-for-net/issues/12211 -->
+    <ProjectReference Remove="$(MSBuildThisFileDirectory)Azure.Management.*\**\*.csproj" Condition="'$(AZURE_KEYVAULT_TEST_MODE)' == 'Live'" />
```
Comment (weshaggard) [inline sdk/keyvault/service.projects:4]: Actually can we scope this to just the test projects?

Comment (heaths) [reply sdk/keyvault/service.projects:4]: I could, but there is only the one (and would likely only ever be one). Care enough for me to push a change now that test runs (playback and live) have started?

**Comment by weshaggard** [comment]
FYI @erich-wang

**Pull request merged**